### DISCUSSION TITLE
Error 404 while deploying

### DISCUSSION THREAD

**Discussion by hemang-h** [vercel/community#400] [Help] [answered]
Hi guys, I am getting an error while trying to deploy my web app using vercel. The error states: 
`404: NOT_FOUND
Code: NOT_FOUND
ID: bom1::6tqfg-1646604864685-5e33085df1cd`

or

![image](https://user-images.githubusercontent.com/73999607/156944826-fd7cd8db-76bd-4d95-8321-1e8ba4914237.png)

My code is running very smoothly and efficiently on my localhost, so I don't seem why this is happening with deploying. 

"_[Click here to learn more about this error,](https://vercel.link/404)_ " does not helped either.

Link to my github project is:  https://github.com/hemang-h/Minting-website-threejs/tree/master

Any help will be appreciated

**Comment by samcx** [comment]
Is it possible to share with us your Deployment URL?

**Comment by Pieparker** [comment, marked as answer]
Hey there! Looks like the deployment has since been removed/deleted. However, in future, if you're seeing this error, it's likely because of the directory config within the UI. Your `output directory` should be the directory which would contain your index page. You can learn more about this [here](https://vercel.com/docs/concepts/deployments/build-step#output-directory).
